### PR TITLE
AAF-73 Refactor Input to include html input attribute list

### PR DESCRIPTION
**Pull request by jwu910**

### CODE REVIEW

**Comment by jwu910** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by rainrivas** [comment]
@jwu910 so what's the problem with travis now
`/home/travis/build/5-gwoap/adopt-a-family/src/components/Form.js
  53:12  error  'formTitle' is missing in props validation  react/prop-types`

**Comment by rainrivas** [comment]
Looks like we only have snapshot errors. @psyduc @phuchle review and it and merge if you can. Otherwise I will do so mañana

**Review by jwu910 — COMMENTED** [review]
No wonder... goodness

File: src/components/Form.js [hunk 1]
```diff
@@ -102,20 +102,20 @@ class Form extends React.Component {
                 name="zipcode"
               />
 
-              <Select
+              <Input
                 animate={animate}
-                form="registration"
                 label="State"
-                menuType="states"
+                menuType={STATES}
```
Comment (phuchle) [inline src/components/Form.js]: From looking at just this code, it's not obvious to me what the menuType prop does, so I think we would do well to rename it to something else that is more descriptive.  Maybe `menuData` or `dropdownData`.

**Review by phuchle — CHANGES_REQUESTED** [review]

**Review by phuchle — APPROVED** [review]

**Review by rainrivas — APPROVED** [review]

**Pull request merged**